tests/test_routes.py

```diff
@@ -556,20 +558,45 @@ def test_delete_item_not_found(self):
     def test_delete_shopcart_item_with_none_shopcart(self):
```

Comment:
Found most part of this test case was removed accidentally when resolving the conflict.

Reply:
Hence I added a commit to bring it back lol